Source/DafnyLanguageServer/Handlers/DafnyTextDocumentHandler.cs

```diff
@@ -106,7 +106,6 @@ public class DafnyTextDocumentHandler : TextDocumentSyncHandlerBase {
         notificationPublisher.HideDiagnostics(documentId);
       } catch (Exception e) {
         telemetryPublisher.PublishUnhandledException(e);
-        logger.LogError(e, "error while closing the document");
```

Comment:
I'm not sure about that, why did you remove this log? Is it because the logger is not available after refactoring?

Reply:
It's moved into PublishUnhandledException so other locations would also do this logging